.github/workflows/release.yml

```diff
@@ -0,0 +1,185 @@
+name: Release CLI Plugins
+
+on:
+  push:
+    branches: [main]
```

Comment:
@shafeeqd959 Quick query, As per the current release plan we publish from dev and we merging to main. I can see the main branch here. In that case, we'll merge/push final changes to the main branch directly and workflow will take care of the release.?

Currently, If something goes wrong we'll test and verify and publish again fix version, then we merge it to the main branch. We trying to make sure the main branch always should have proper fix/working code I guess

CC @netrajpatel

Reply:
We always release the code of main and that should be proper. To make sure the stability of the code, we are updating the test suite which is listening on the dev branch changes.